src/components/Avatar/Avatar.tsx

```diff
@@ -18,11 +18,16 @@ export const AVATAR_CLASS_SET = [
 ];
```

Comment:
expose theme set as "theme": type : OcThemeNames(src/components/ConfigProvider/Theming/Theming.types.ts)
prop randomiseTheme: if true, use Math.floor(Math.random() * 100) % AVATAR_CLASS_SET.length;
passing the hashing function as prop